tests/test.bats

```diff
@@ -258,3 +263,31 @@ _healthcheck_wait ()
 	fin rm -f
 	rm -f .docksal/docksal-local.env
 }
+
+@test "Check Platform.sh Integration" {
+	[[ $SKIP == 1 ]] && skip
+
+	### Setup ###
+	docker rm -vf "$NAME" >/dev/null 2>&1 || true
+	docker run --name "$NAME" -d \
+		-v /home/docker \
+		-v $(pwd)/../tests:/var/www \
+		-e SECRET_PLATFORMSH_CLI_TOKEN \
+		"$IMAGE"
+	_healthcheck_wait
+
+	### Tests ###
+	run docker exec -it -u docker "$NAME" bash -c 'source $HOME/.docksalrc >/dev/null 2>&1; echo "${SECRET_PLATFORMSH_CLI_TOKEN}"'
+	run fin exec 'echo ${PLATFORMSH_CLI_TOKEN}'
+	[[ "${output}" != "" ]]
+	unset output
+
+	run docker exec -it -u docker "$NAME" bash -c 'source $HOME/.docksalrc >/dev/null 2>&1; platform auth:info'
```

Comment:
Even though this is a useful test, it relies on personal credentials, which may get invalidated at some point, which will break tests. I think it should be dropped.

Reply:
@lmakarov I disagree and believe that we should be testing it. Especially because we want to make sure that it doesn't break. I believe that maybe we should have a general set of credentials for Docksal rather then my own set of credentials but if I didn't use my own personal credentials I wouldn't have been able to debug what was happening and if the `SECRETS_` conversion piece.

If we are going to offer such services within the containers then I believe we need to at least test to make sure they login and response accordingly.

Reply:
@sean-e-dietrich ok, sounds reasonable. Does platform.sh offer a free tier?
We'll need an account registered with developer[@]docksal.io

Also, if we treat Platform.sh integration this way, then, to be fair, we need accounts with Pantheon and Acquia as well and update those tests :)

Reply:
@lmakarov yes, Platform has the ability to create an account but nothing has to be created for that account. I also agree about the Pantheon and Acquia tests.